deployment/terraform/assets/cluster.tf

```diff
@@ -84,6 +84,7 @@ resource "aws_instance" "app_server" {
       "sudo apt-get install -y mysql-client-8.0",
       "sudo apt-get install -y postgresql-client-14",
       "sudo apt-get install -y prometheus-node-exporter",
+      # "sudo NEEDRESTART_MODE=a apt-get install -y redis-tools",
```

Comment:
What's the deal with this one?

Reply:
This is needed to install the `redis-cli`. But the thing is there is an UI intervention while installing, which is supposedly fixed with `NEEDRESTART_MODE=a`. But that still somehow didn't fix things. So just ignoring for now. If someone needs to connect to Redis, they can install manually.

Reply:
FWIW, the command worked fine on my end (EC2 running Ubuntu `22.04`).

Reply:
Hmm .. weird. I will test it again later.